.gitignore

```diff
@@ -63,3 +63,6 @@ override.tf.json
 # Ignore CLI configuration files
 .terraformrc
 terraform.rc
+
+proto-public/*/.ratelimit
+proto/*/.ratelimit
```

Comment:
I'm ok with using `.ratelimit` as a suffix but I would prefer something more generic like `.tmp` or `.temp` so we don't end up with a list of suffixes in the future.